constellation-app/constellation
# Remove annotation for returning tuple

### Prerequisites


    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change


Removed an annotation in `constellation_client`. This was causing issues on the Linux machine, because it was running an old version of python. To remedy, a better description of the function and the expected return types were added in a comment above the function.

### Alternate Designs


N/A

### Why Should This Be In Core?


N/A

### Benefits


Script works on old python versions

### Possible Drawbacks


None

### Verification Process



1. Open consty,
2. Tools -> Start Jupyter Notebook
3. Import constellation_client
4. Observe no errors are thrown when importing

### Applicable Issues


https://github.com/constellation-app/constellation/issues/2176
